app/assets/stylesheets/ama_layout/layout_components/option-toggle.scss

```diff
@@ -0,0 +1,52 @@
+.option-toggle{
+  &__list{
+    list-style: none;
+    margin-bottom: 0;
+  }
+
+  &__item{
+    @include clearfix;
+    margin: 1px 0;
+    background-color: $white;
+    color: $cerulean;
+    position: relative;
+
+    &:nth-child(even){
+      background-color: $ash;
+    }
+
+    &.visible{ //this class isn't BEM because of the old JS in RR - KR
+      background-color: $brand-blue-light;
+      color: $white;
+
+      .option-toggle__item-icon{
+        background-color: $brand-blue-light;
+        color: $white;
+      }
+    }
+  }
+
+  &__item-label{
```

Comment:
do these need to be attached to the item? or could they just be option-toggle__label, option-toggle__icon-container, etc?